addons/l10n_id_efaktur/models/account_move.py

```diff
@@ -176,15 +176,15 @@ def _generate_efaktur_invoice(self, delimiter):
             eTax['NAMA'] = move.partner_id.name if eTax['NPWP'] == '000000000000000' else move.partner_id.l10n_id_tax_name or move.partner_id.name
             eTax['ALAMAT_LENGKAP'] = move.partner_id.contact_address.replace('\n', '') if eTax['NPWP'] == '000000000000000' else move.partner_id.l10n_id_tax_address or street
             eTax['JUMLAH_DPP'] = int(float_round(move.amount_untaxed, 0)) # currency rounded to the unit
-            eTax['JUMLAH_PPN'] = int(float_round(move.amount_tax, 0))
+            eTax['JUMLAH_PPN'] = int(float_round(move.amount_tax, 0, rounding_method="DOWN"))  # tax amount ALWAYS rounded down
             eTax['ID_KETERANGAN_TAMBAHAN'] = '1' if move.l10n_id_kode_transaksi == '07' else ''
             eTax['REFERENSI'] = number_ref
             eTax['KODE_DOKUMEN_PENDUKUNG'] = '0'
 
             lines = move.line_ids.filtered(lambda x: x.product_id.id == int(dp_product_id) and x.price_unit < 0 and not x.display_type)
             eTax['FG_UANG_MUKA'] = 0
-            eTax['UANG_MUKA_DPP'] = int(abs(sum(lines.mapped(lambda l: float_round(l.price_subtotal, 0)))))
-            eTax['UANG_MUKA_PPN'] = int(abs(sum(lines.mapped(lambda l: float_round(l.price_total - l.price_subtotal, 0)))))
+            eTax['UANG_MUKA_DPP'] = abs(float_round(sum(lines.mapped(lambda l: float_round(l.price_subtotal, 2))), 0))
+            eTax['UANG_MUKA_PPN'] = abs(float_round(sum(lines.mapped(lambda l: float_round(l.price_total - l.price_subtotal, 2))), 0))
```

Comment:
As said on Discord => it is generally rounded to 2 dp.
But the settings for the Rupiah is actually rounding to 2dp too by default
![image](https://github.com/odoo/odoo/assets/66360871/97435401-5429-4b54-a748-f49d5472d9bd)

So in my opinion, we should use the currency and let it do the rounding to x decimal place so that it matches the user configuration (unless there is a specific requirement in the efaktur documentation saying that it should be rounded to 2 decimals all the time, I don't know where to find this doc)
Also, some of these are rounded to 4 dp. I don't see anywhere in the task where it is asked so could you please add the reason for it as a comment if it is correct? Thanks

Reply:
@nni-odoo Does it make sense to have the rounding be 0.01 in Indonesia given the super low value of the currency ?
Maybe something to check with ELON

Reply:
I've talked to ELON (the FC submitting the ticket) and he agrees with the rounding to match currency's decimal place!